observation_portal/accounts/test_utils.py

```diff
@@ -1,18 +1,25 @@
 from mixer.backend.django import mixer
 from django.utils import timezone
+from django.db.models import signals
+from django.db.models.signals import post_save
 
 from observation_portal.accounts.models import User, Profile
+from observation_portal.common.test_helpers import disconnect_signal
+from observation_portal.accounts.signals.handlers import cb_user_post_save, cb_profile_post_save
 
 
 def blend_user(user_params=None, profile_params=None):
```

Comment:
This method is now used in most tests to blend the User/Profile, since it disables the new signals before creating so we don't gum up the tasks for those tests that depend on the tasks.

Reply:
Nice!